tests/parsers/pipe.parser.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('PipeParser', () => {
 	});
 
 	it('should extract strings with escaped quotes', () => {
-		const contents = `Hello {{ 'World\'s largest potato' | translate }}`;
+		const contents = `Hello {{ 'World\\'s largest potato' | translate }}`;
```

Comment:
Hmm, you're right, but using double backslash makes AOT choke:

```
[08:45:07]  ngc started ...
[08:45:13]  ionic-app-script task: "build"
[08:45:13]  Error: Template parse errors: Parser Error: Unexpected token 'text' at column 22 in [ {{ 'Categories test\\'
            text' | translate }} ] in ShopPage@5:37 (" <button ion-button icon-left (click)="onShowCategories()">
            <ion-icon name="list"></ion-icon>[ERROR ->] {{ 'Categories test\\' text' | translate }} </button>
```

Not the fault of your PR, just wondering what the correct way to escape quotes is then.